config/initializers/bigquery.rb

```diff
@@ -0,0 +1,12 @@
+require 'google/cloud/bigquery'
+
+BIG_QUERY_API_JSON_KEY = Settings.google.bigquery.api_json_key
+
+if BIG_QUERY_API_JSON_KEY.present?
```

Comment:
Would it make more sense to use the feature flag as the condition here?

```suggestion
  if FeatureService.enabled?(:send_request_data_to_bigquery)
```